lessons/AI/ai-intro/langs/en-us/index.adoc

```diff
@@ -126,6 +126,115 @@ An example of Machine Learning would be the particular set of algorithms that de
+
+[cols="1a,1a,1a", grid="none", frame="none"]
+|===
+|@n @image{images/adirondacks.png}
+|@n @image{images/sunset-mountains.png}
+|@n @image{images/snowy-mountains.png}
+|@n @image{images/grassy-mountains.png}
+|@n @image{images/nz-mountains.png}
+|@n @image{images/sunny-grass-mountains.png}
+|@n @image{images/bear-lake.png}
+|@n @image{images/sunrise-mountains.png}
+|@n @image{images/arizona-mountains.png}
+|===
+}
+
+@QandA{
+@Q{What similarities do the 9 photos have?}
+@A{Responses will vary.}
+@A{They all show mountains and sky.}
+@A{The photos are all rectangular in shape, and the rectangles are all the same size.}
+@A{The photos all make me want to visit the mountains.}
+
+@Q{What differences do the 9 photos have?}
+@A{Some photos show a body of water, others do not.} @A{Some photos include the sun, others do not.}
+@A{Some photos show mountains with snow, others show mountains with grass.} 
+@A{The color palettes differ from photo to photo (blues, greens, whites).}
+@A{Some mountains appear round while others appear pointy.}
+@A{In some photos, the mountains occupy most of the photograph, while in others, the mountains are just a small fraction of what is represented.}
+}
+
+
+@lesson-instruction{
+Complete the *second section* of @printable-exercise{quantifying-similarity.adoc}.
+}
+
+@QandA{
+@Q{How was the experience of identifying similar and different photos? Was it simple or challenging?}
+@A{Student responses will vary. Some students may articulate that it was challenging to decide which similarities to prioritize, when they were able to identify similarities across all photos.}
+
+@Q{How confident are you in the decisions you made on the second section of the @printable-exercise{quantifying-similarity.adoc}?} 
+@A{Student responses will vary.}
+}
+
+
+Let's think about another way of quantifying similarity. 
+
+@center{@image{images/mountains-v-water.png, 400}}
+
+@QandA{
+
+@Q{What do you Notice about the coordinate plane image?}
+@A{Only the first quadrant is visible.}
+@A{The x-axis is "size of body of water".}
+@A{The y-axis is "size of mountains".}
+@A{There are three mountain landscapes (numbers 1, 5, and 7) positioned at various points on the coordinate plane.}
+
+@Q{What do you Wonder about it?}
+@A{How were the labels on the axes determined?}
+@A{Where would I put the other landscapes on this coordinate plane?}
+@A{How can I position landscapes that do not have any body of water in them?}
+
+@Q{Where do you think Photo 2 belongs on this coordinate plane?}
+@A{Invite a student to the board to identify where Photo 2 goes.}
+@A{It should have a similar y-coordinate and a slightly greater x-coordinate.}
+}
+
+@lesson-instruction{
+Complete the third section of @printable-exercise{quantifying-similarity.adoc}.
+}
+
+NEED TO HAVE SOME DISCUSSION OF WHAT THEY JUST DID / LEARNED / WHAT'S IMPORTANT.
```

Comment:
@retabak Or maybe what feels like it's missing from the first synthesize question will happen here?